lib/ansible/plugins/callback/__init__.py

```diff
@@ -432,6 +432,9 @@ def _clean_results(self, result, task_name):
                 # 'var' value as field, so eliminate others and what is left should be varname
                 for hidme in self._hide_in_debug:
                     result.pop(hidme, None)
+                for control_var in ("ansible_loop_var", "ansible_index_var", "ansible_loop"):
+                    if control_var in result:
+                        result.pop(control_var, None)
```

Comment:
While not technically wrong, the use of `pop()` with a default is redundant with first checking whether the key exists. Personally I would remove the condition to make it match the previous loop (or combine the two), but if you prefer the condition you should just do `result.pop(control_var)`.

Reply:
Make sense.